### DISCUSSION TITLE
http代理Chromecast，Youtube Netflix无法打开

### DISCUSSION THREAD

**Discussion by Jiematcha** [v2fly/v2ray-core#676] [Q&A] [answered]
仅使用v2ray core 非v2rayN
{
      "listen": "0.0.0.0",
      "port": 8080,
      "protocol": "http",
      "settings": {
        "timeout": 100,
        "allowTransparent": false,
        "userLevel": 0},
      "streamSettings": {},
      "tag": "HTTP inbound",
      "sniffing": {
          "enabled": true,
          "destOverride": [
              "http",
              "tls"
          ]
      },
      "allocate": {
          "strategy": "always"
      }
}
同样的http代理给手机可以正常使用Youtube
http代理下Chromecast上的国内应用可以使用
Chromecast访问 Netflix 403
Youtube显示无互联网连接
然而系统没有提示无互联网连接，还能正常访问Google Play

**Comment by aoterbear** [comment, marked as answer]
chromecast上youtube和netflix这种app是没办法走系统的proxy的。
